src/MaterialDesignThemes.Wpf/Converters/DialogBackgroundBlurConverter.cs

```diff
@@ -8,7 +8,7 @@
 using System.Windows.Media.Effects;
 
 namespace MaterialDesignThemes.Wpf.Converters;
-internal sealed class DialogBackgroundBlurConverter : IMultiValueConverter
+public sealed class DialogBackgroundBlurConverter : IMultiValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.